packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -526,26 +537,25 @@ export function makeWallet({
             })
             .map((payoutP, payoutIndex) => {
               const keyword = payoutIndexToKeyword[payoutIndex];
```

Comment:
While we're in here fixing this, using a `payoutIndex` seems like vestigial code. I think we can clean it up by just iterating over the payoutObj directly.

Reply:
Gotcha.  Done.